modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+
+As a consequence, when backspacing from an empty block from another empty block within {productname}, the editor would lose the preserved formatting for the previous block element.
+Its worth noting that `_this action is only excpected for when content is inserted into a `br[data-mce-bogus]` empty block. This is because the br[data-mce-bogus] tag is programmed to reset the formatting to the default setting for a empty block such as the `br`.
+
+[TIP]
+To avoid this, it is *important* to use a `backspace` instead of an `enter key` when moving from one empty block to another. This will ensure that the formatting of the previous block is preserved and the `br[data-mce-bogus]` tag is not inserted.
+
+To fix this issue, {productname} added improvements that now allow merge blocks to filter all block element decentants, such as the `br[data-mce-bogus]` block element, to ensure preservation of previously applied formatting is not removed from other block elements.
+
+In {productname} 6.4, pressing backspace in an empty line now preserves formatting in a previous empty line.
+
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+=== Pressing enter inside the inputfontsize input would not move the focus back into the editor content
+
+// https://ephocks.atlassian.net/browse/TINY-9598
+During development for the new `fontsizeinput` toolbar item, an issue was found that by pressing enter inside the input field would not move the `focus` back into the {productname} editor body. 
+
+As a consequence, any changes made to the font size within the new toolbar input field, would result in the user having no way to use the new font size within the editor.
+
+After some improvements to the `fontsizeinput` toolbar item, pressing enter in the input field now sets the focus back to the {productname} editor viewport, which allows the new font size to be used. 
+
+In {productname} 6.4, pressing enter inside the `inputfontsize` input now returns the focus back to the editor content body.
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+
+[[additions]]
+== Additions
+
+{productname} 6.4 also includes the following additions:
+
+=== New tree component that can be used in dialog body panel
+
+// https://ephocks.atlassian.net/browse/TINY-9532
+A new tree component has been created to support the *Advanced Template* premium plugin that works in conjuction with a new tree dialog body panel.
+
+This tree component is limited to *one* level and is used in the dialog body panel to allow users to organize templates into levels for better UI. The tree component is designed to create a visual representation of the hierarchical structure for the template system, which allows users to easily identify the various templates and levels within each catagory, giving users the ability to quickly and easily navigate through the templates, and select the right one for their needs.
```

Comment:
```suggestion
This tree component is limited to *one* level and is used in the dialog body panel to allow users to organize templates into levels for better UI. The tree component is designed to create a visual representation of the hierarchical structure for the template system, which allows users to easily identify the various templates and levels within each category, giving users the ability to quickly and easily navigate through the templates, and select the right one for their needs.
```

Reply:
@hamza0867 Thank you for the above, I have put together a new suggestion to replace the existing paragraph.
Closing this dialog so we can consolidate the feedback in one section.

Reply:
@kemister85 where is this new paragraph please ?

Reply:
Closing this conversation as it has been resolved in the new reviewed suggestions.